roles/openshift_storage_nfs/handlers/main.yml

```diff
@@ -4,3 +4,8 @@
     name: nfs-server
     state: restarted
   when: not (nfs_service_status_changed | default(false))
+
+- name: restart nfs-config
+  service:
+    name: nfs-config
```

Comment:
Might want to just restart nfs-config in the playbook if the config changes rather than waiting for the trigger...

That or not start the nfs-server in tasks/main.yml and just enable it and let the restart handler handle starting the service.